core/crypto/typed_key_storage/typed_key_storage_impl.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/typed_key_storage/typed_key_storage_impl.hpp"
+namespace kagome::crypto::storage {
+
+  // It's not clear yet what keys should be returned
+  TypedKeyStorageImpl::EDKeys TypedKeyStorageImpl::getEdKeys(
+      KeyTypeId key_type) {
+    EDKeys keys;
+    auto &map = ed_keys_[key_type];
```

Comment:
Pls, use `.at()` or `find()` methods for trying to get element of map.
`.operator[]()` instantiate new element if that not exists before.